canvas_admin_tools/settings/base.py

```diff
@@ -136,6 +139,22 @@
     }
 }
 
+# RQ
+# http://python-rq.org/docs/
+
+RQ_QUEUES = {
+    'default': {
+        'HOST': REDIS_HOST,
+        'PORT': REDIS_PORT,
+        'DB': 0,
```

Comment:
Thanks Dave. Yes, that's as far as I got, and I'm not sure what that means, since redis isn't typically backended with a database...? I think I have a lot of reading to do to understand this end to end.